packages/babel-plugin-transform-es2015-modules-commonjs/test/fixtures/spec/exports/expected.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+module.exports = exports = Object.create(null, {
+  __esModule: {
+    value: true
+  }
+});
+const _undefined = {
+  enumerable: true,
+  configurable: true,
```

Comment:
Oops, I guess these (and the regular records as well) should be `writable: true` instead.

While the export record itself is defined to not be `writable`, when observed from an importing module, it is defined to be `writable: true`.